server/auth/handlers/handler.go

```diff
@@ -337,11 +338,13 @@ func (h *AuthHandlers) enrichTmplData(req *request.AuthReq) interface{} {
 	if req.AuthUser != nil {
 		// fetch current user with updated fields
 		user, err := h.UserService.FindByAny(req.Context(), req.AuthUser.User.ID)
-		if err != nil {
-			return nil
+
+		// check if err is not nil and cater for MFA by checking if the error is not allowed to read
+		if err != nil && !errors.Is(err, service.UserErrNotAllowedToRead()) {
```

Comment:
what happens when there is a `UserErrNotAllowedToRead` error?

Reply:
In this case, since we have the User details from `req.AuthUser` and we're searching for the most updated `users` data, the  UserErrNotAllowedToRead is ignored.

Reply:
Unless maybe we don't want the current user's theme to be applied on MFA login screen, then we can handle it the same way we handle other errors.